spec/factories/one_time_tokens.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: one_time_tokens
+#
+#  cis2_info  :jsonb            not null
+#  jsonb      :jsonb            not null
+#  token      :string           not null, primary key
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  user_id    :bigint           not null
+#
+# Indexes
+#
+#  index_one_time_tokens_on_created_at  (created_at)
+#  index_one_time_tokens_on_token       (token) UNIQUE
+#  index_one_time_tokens_on_user_id     (user_id) UNIQUE
+#
+# Foreign Keys
+#
+#  fk_rails_...  (user_id => users.id)
+#
+FactoryBot.define do
+  factory :one_time_token, class: ReportingAPI::OneTimeToken do
```

Comment:
```suggestion
  factory :reporting_api_one_time_token do
```

What do you think about using this name everywhere?

Reply:
sure, done